docs/resources/resources.md

```diff
@@ -0,0 +1,39 @@
+---
+id: resources
+title:  "Resources"
+---
+
+## Blog Articles
+
+Note that these articles reflect the state of ZIO at the time of their publication, so the code samples might be outdated considering ZIO is still under active development.
+- [Bifunctor IO: A Step Away from Dynamically-Typed Error Handling](http://degoes.net/articles/bifunctor-io) by John De Goes
+- [Beautiful, Simple, Testable Functional Effects for Scala](http://degoes.net/articles/zio-environment) (introducing ZIO Environment) by John De Goes
+- [Testing Incrementally with ZIO Environment](http://degoes.net/articles/testable-zio) by John De Goes
+- [ZIO & Cats Effect: A Match Made in Heaven](http://degoes.net/articles/zio-cats-effect) by John De Goes
+- [Elevator Control System using ZIO](https://medium.com/@wiemzin/elevator-control-system-using-zio-c718ae423c58) by Wiem Zine Elabadine
```

Comment:
There is also:

<http://justinhj.github.io/> (Hacker News 5, and Using Fetch with ZIO)

<https://blog.softwaremill.com/scalaz-8-io-vs-akka-typed-actors-vs-monix-part-1-5672657169e1> (Part 1, 2, 3 — Bit dated by now)

<https://abhsrivastava.github.io/2018/11/03/Hangman-Game-Using-ZIO/> (Building Hangman Game Using ZIO)

I would remove "Scalaz" from any blog post titles, e.g. "Scalaz Stream" => "ZIO Stream", just to avoid confusing people.

There is a series of blog posts in Japanese on ZIO, but I can't find them at the moment...

Reply:
Thanks! I'm gonna add month/years for talks and articles and sort them to have the most recent first